.gitattributes

```diff
@@ -1,3 +1,7 @@
 # Ensure Docker script files uses LF to support Docker for Windows.
-setup_docker_prereqs    eol=lf
-/virtualization/Docker/scripts/*    eol=lf
\ No newline at end of file
+# Ensure "git config --global core.autocrlf input" before you clone
+setup_docker_prereqs                text eol=lf
+/virtualization/Docker/scripts/*    text eol=lf
+/script/*                           text eol=lf
+*.sh                                text eol=lf
```

Comment:
There are no shell or Python scripts in the root? or does this also match `**/*.sh` ?

Reply:
`*.py` matches all `**/*.sh`:
```
λ git check-attr --all homeassistant\scripts\db_migrator.py
"homeassistant\\scripts\\db_migrator.py": text: set
"homeassistant\\scripts\\db_migrator.py": eol: lf
"homeassistant\\scripts\\db_migrator.py": whitespace: error
```